index.src.html

```diff
@@ -534,10 +535,13 @@ <h2 id="algorithms">Algorithms</h2>
 
   <h3 id="parsing">Parsing</h3>
 
-  <h4 id="get-types">
+  <h4 id="algorithm-get-types" algorithm="get types">
     Which data types ought to be removed for |response|?
   </h4>
 
+  The |types| which ought to be removed are determined as the result of running
+  the <dfn>get types</dfn> algorithm on |response|.
```

Comment:
I'd prefer that we use the same pattern here as elsewhere:

"`Given a <a>response</a> (|response|), the following algorithm returns a list of storage types which are specified in |response|'s Clear-Site-Data header."